src/drivers/uavcan_v1/Uavcan.hpp

```diff
@@ -113,7 +113,7 @@ class UavcanNode : public ModuleParams, public px4::ScheduledWorkItem
 	* Current size of 8192 bytes is arbitrary, should be optimized further
 	* when more nodes and messages are on the CAN bus
 	*/
-	static constexpr unsigned HeapSize = 8192;
+	static constexpr unsigned HeapSize = 2 * 8192;
```

Comment:
This should be configurable, either board-wise see #17100, or using a paramter. Our FMUK66 is on low on free RAM.

Reply:
Oh right, I didn't mean to commit this; I was playing around with code and at one point noticed I had run out of RAM so I increased it.  I'll revert this.

Reply:
Fixed